docs/GettingStarted.md

```diff
@@ -479,6 +479,21 @@ for the first time:
 Remember that the debug mode may affect your application performance and so it must not be used
 in a production environment.
 
+### Using a custom logger
+
+By default, all logs are processed by the default Ruby logger.
+Typically, when using Rails, you should see the messages in your application log file.
+Datadog client log messages are marked with ``[ddtrace]`` so you should be able
+to isolate them from other messages.
+
+Additionally, it is possible to override the default logger and replace it by a
+custom one. This is done using the ``log`` attribute of the tracer.
+
+    buf = StringIO.new                            # Log messages should go there
+    Datadog::Tracer.log = Logger.new(buf)         # Overriding the default tracer
+    Datadog::Tracer.log.info { "this is typically called by tracing code" }
+    puts buf.string                               # Print all our custom log content
```

Comment:
should we use the `buf.string` example? isn't better to provide a base example to output to a different file?